src/transaction.ts

```diff
@@ -181,4 +182,40 @@
     this.postConditions = LengthPrefixedList.deserialize(bufferReader, PostCondition);
     this.payload = Payload.deserialize(bufferReader);
   }
+
+/**
+ * Broadcast the signed transaction to a core node
+ *
+ * @param {String} apiURL - specify the core node URL to broadcast to
+ * 
+ * @returns {Promise} that resolves to a response if the operation succeeds
+ */
+  broadcast(apiURL?: string) {
```

Comment:
Should this function include any "damage control" features, such as only being able to fire once per instance unless a modifying parameter is included?

I imagine if someone left this in a loop by accident it could be pretty detrimental.

Reply:
Yes that's why a nonce is required for every transaction. The nonce needs to be incremented for every transaction. Otherwise the transaction is invalid and will be rejected by the core node.